telemetry/specs/INT.mdk

```diff
@@ -1335,7 +1352,7 @@ be accessed at the link below:\
       is *excluded* from the *Hop ML*.
     - The largest value of *Hop ML* for baseline and domain specific
       metadata is 31.
-    - An INT-capable device may be limited in the maximum number
+    - An INT node may be limited in the maximum number
```

Comment:
Do we consider the case where a node is not capable of processing a specific instruction, while it's not limited by the max number of instructions or metadata?

Reply:
I think so. Line 1359-1366 states

An INT hop that cannot process all instructions has the following alternatives:
      - Send to the monitoring system *Hop ML* \* 4 bytes, with all-ones reserved
        value (4 or 8 bytes of 0xFF depending on the length of metadata) for the
        metadata corresponding to instructions it cannot process, or
      - Send to the monitoring system the metadata it can process, update the
        *RepMdBits*, *DSMdBits* and *MD Length* fields appropriately in the
        Telemetry Report.